src/ngx_http_lua_socket_tcp.c

```diff
@@ -1980,12 +1960,14 @@ ngx_http_lua_socket_prepare_error_retvals(ngx_http_request_t *r,
     u_char           errstr[NGX_MAX_ERROR_STR];
     u_char          *p;
 
-    if (ft_type & (NGX_HTTP_LUA_SOCKET_FT_RESOLVER
-                   | NGX_HTTP_LUA_SOCKET_FT_SSL))
-    {
+    if (ft_type & NGX_HTTP_LUA_SOCKET_FT_RESOLVER) {
         return 2;
     }
 
+    if (ft_type & NGX_HTTP_LUA_SOCKET_FT_SSL) {
+        return 0;
```

Comment:
Unclear about what the return values for this function mean. Would it make sense to replace the `0` and the `2` by names, like `NGX_AGAIN` and `NGX_ERROR`?

Reply:
This is the number of return values to the caller of the function. It is not a special constant.